src/pybamm/experiment/step/step_termination.py

```diff
@@ -14,8 +14,11 @@ class BaseTermination:
         The value at which the event is triggered
     """
 
-    def __init__(self, value):
+    def __init__(self, value, operator=None):
         self.value = value
+        if operator not in ["<", ">", None]:
+            raise ValueError(f"Invalid operator: {operator}")
+        self.operator = operator
```

Comment:
This could be a good case for using an enum. Then arbitrary stings can't be passed in and type checkers can see the error